src/tracks/timetrack/ui/TimeTrackView.cpp

```diff
@@ -156,9 +158,26 @@ void TimeTrackView::Draw(
    const wxRect &rect, unsigned iPass )
 {
    if ( iPass == TrackArtist::PassTracks ) {
+      const auto pTrack = FindTrack();
+      auto pList = pTrack->GetOwner();
+      if (!pList)
+         // Track isn't owned by a list.  Can't proceed!
+         return;
+      if (!pList)
+         // List isn't owned by a project.  Can't proceed!
+         // But this shouldn't happen when drawing it
+         return;
```

Comment:
One of these checks is redundant, or maybe something is missing here?

Reply:
Thanks I think I meant `!pList->GetOwner()`